plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesExt.kt

```diff
@@ -219,13 +219,23 @@ fun MapPluginExtensionsDelegate.setGesturesManager(
 }
 
 /**
- * The gesture configuration object.
+ * Get copy of current gesture settings.
+ *
+ * Updating any property of [GesturesSettings] object returned will not take any effect,
+ * please use MapView.gestures.updateSettings { } instead.
+ *
  * Gesture plugin with id = [Plugin.MAPBOX_GESTURES_PLUGIN_ID] must be added while constructing
  * [com.mapbox.maps.MapView] as part of [com.mapbox.maps.MapInitOptions.plugins].
  *
  * @throws IllegalStateException if gestures plugin was not added.
  * @see [com.mapbox.maps.MapInitOptions]
  */
+@Deprecated(
+  "Gesture plugin instance obtained from MapView should be used to get particular setting e.g. mapView.gestures.focalPoint",
+  replaceWith = ReplaceWith(
+    "mapView.gestures.desiredSetting()",
```

Comment:
Found the only way like that to actually replace at least smth with hint from IDE.

Reply:
I think the exact replacement of this function(get a copy of current gesture settings) is `mapView.gestures.getSettings()`. However we could keep the message to instruct the way to modify a setting.

Reply:
```suggestion
    "mapView.gestures.getSettings()",
```